### DISCUSSION TITLE
IOException: Error loading the model: XML Error: problem reading attribute 'range' Element 'joint', line 1

### DISCUSSION THREAD

**Discussion by kaizu3** [google-deepmind/mujoco#2030] [Asking for Help]
Hi,

I'm a (student / professor / engineer) and I'm trying to use MuJoCo for a project.

I'm looking for some help with this error when I import the humanoid mujoco scene or any mujococ scene in unity I get this error ""IOException: Error loading the model: XML Error: problem reading attribute 'range' Element 'joint', line 1 "".

Here is a screenshot / video, illustrating my question:
![image](https://user-images.githubusercontent.com/130810535/235503934-acc6ccd1-31af-4b2d-ad0b-198eea8f08dd.png)

Please

**Comment by erez-tom** [comment]
If you input a path to the text box called "Debug File Name" (at the top of the "Mj Global Settings" component) you can save the XML that's generated from your scene.  You can then inspect it (in particular, search for mentions of "range") and see which element is causing the problem.  Is this a fresh import from the humanoid XML, or a scene/prefab you've been modifying?

**Comment by kaizu3** [comment]
It is a fresh import and i tried multiple models from different sources and versions of mujoco and i get the same error.
I tried it in mujoco and it worked just fine.
I linked the xml debug file if you can see any problem.
Thank you very much for offering to help.

[HUMANOID.txt](https://github.com/deepmind/mujoco/files/11368750/HUMANOID.txt)

**Comment by stonfute** [comment]
It seems more related to `CultureInfo`, where a float like `75.00` is converted to `75,00` in some countries (I'm french)

I'm currently facing the same problem where range is wrongly converted in my debug XML `range="-75,00018 75,00018"`

Funny fact, every other float value is well converted. 

It should use `MjEngineTool.MakeLocaleInvariant($"{RangeLower} {RangeUpper}"));`
